src/neptune/common/hardware/constants.py

```diff
@@ -13,7 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-__all__ = ["BYTES_IN_ONE_MB", "BYTES_IN_ONE_GB"]
+__all__ = ["BYTES_IN_ONE_MB", "BYTES_IN_ONE_GB", "WATTS_TO_MILLIWATTS"]
```

Comment:
Could you rename it to `MILIWATTS_IN_WATT` instead? For readability and to keep the pattern 😉